features/admin.console-settings.v1/components/console-administrators/administrators-list/administrators-table.tsx

```diff
@@ -448,7 +448,7 @@ const AdministratorsTable: React.FunctionComponent<AdministratorsTablePropsInter
                     data-componentid={ `${ componentId }-empty-placeholder` }
                     image={ getEmptyPlaceholderIllustrations().newList }
                     imageSize="tiny"
-                    subtitle={ [ "There are no collaborator users associated with your organization at the moment." ] }
+                    subtitle={ [ "There are no administrator users associated with your organization at the moment." ] }
```

Comment:
Don't we have i18n support for these?

Reply:
We don't have i18n strings for the empty placeholders. Let's add in a follow up PR.